todo_app/data/session_items.py

```diff
@@ -1,72 +1,150 @@
 from flask import session
+import requests
+import json
 
+import os
 
+boardid = os.environ.get('boardid')
+trello_key = os.environ.get('trello_key')
+trello_token = os.environ.get('trello_token')
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
 
+listtodo = os.environ.get('listtodo')
+listdoing = os.environ.get('listdoing')
+listdone = os.environ.get('listdone')
 
 
 def get_items():
-    """
-    Fetches all saved items from the session.
 
-    Returns:
-        list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
+    Module 2 - amend get_items to use response and Trello api
+    Returns: response json() containing trello card names / id's and lists id's
     """
-    Fetches the saved item with the specified ID.
 
-    Args:
-        id: The ID of the item.
+    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList'
 
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
+    query = {
+        'key': trello_key,
+        'token': trello_token
+            }
+
+    response = requests.request(
+            "GET",
+            url,
+            params=query
+            )
 
+    return make_get_request(url, query)
 
 def add_item(title):
     """
-    Adds a new item with the specified title to the session.
-
+    Makes Trello POST call to create a new card 
+    list predefined as todo hard coded for now
     Args:
-        title: The title of the item.
+        title: The title/name of the item.
 
     Returns:
         item: The saved item.
     """
-    items = get_items()
 
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
+    url = "https://api.trello.com/1/cards"
+
+    query = {
+        'key': trello_key,
+        'token': trello_token,
+        'idList': listtodo,
+        'name': title
+        }
+
+    response = requests.request(
+        "POST",
+        url,
+        params=query
+        )
+
+    
+    return response.json()
+
+def commence_item(id):
+    """
+        makes call to move_card with list id for "doing" list
+    Args:
+        id = card id    
+    """
+    return move_card(id,listdoing)    
 
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
+def get_lists():
```

Comment:
I don't think this is used anywhere

Reply:
earlier thinking on solution , method now removed
plan was to make call to retrieve trello list id's
these are now in the .env file